docs/details/main-components/font.rst

```diff
@@ -45,6 +45,31 @@ To test it try
 If all works well, a ✓ character should be displayed.
 
 
+Typesetting
+***********
+
+Although LVGL can decode and display any Unicode characters
+(assuming the font supports them), LVGL cannot correctly render
+all complex languages.
+
+The standard Latin-based languages (e.g., English, Spanish, German)
+and East Asian languages such as Chinese, Japanese, and Korean (CJK)
+are relatively straightforward, as their characters are simply
+written from left to right.
+
+Languages like Arabic, Persian, and Hebrew, which use Right-to-Left
+(RTL) or mixed writing directions, are also supported in LVGL.
+Learn more :ref:`here <bidi>`.
+
+For characters such as ``é`` ``ü``, ``ő``, ``Á``, and ``Ű``, it is recommended
```

Comment:
```suggestion

.. |Aacute| unicode:: U+000C1 .. LATIN CAPITAL LETTER A WITH ACUTE
.. |eacute| unicode:: U+000E9 .. LATIN SMALL LETTER E WITH ACUTE
.. |otilde| unicode:: U+000F5 .. LATIN SMALL LETTER O WITH TILDE
.. |Utilde| unicode:: U+00168 .. LATIN CAPITAL LETTER U WITH TILDE
.. |uuml|   unicode:: U+000FC .. LATIN SMALL LETTER U WITH DIAERESIS

For characters such as '|eacute|', '|uuml|', '|otilde|', '|Aacute|', and '|Utilde|', it is recommended
```

Reply:
FYI, Sphinx doesn't know what to do with characters outside the ASCII range but does well with the "replacement" directives as above.  Note the in-line literal back-quotes hides the replacement string from the `reStructuredText` parser unfortunately, so the letters have to be enclosed in single- or double-quotes or some other character that doesn't switch the parser into a "different mode".

Reply:
Wao, thank you!